api/types/container/host_config.go

```diff
@@ -313,7 +313,7 @@ type HostConfig struct {
 	Runtime         string            `json:",omitempty"` // Runtime to use with this container
 
 	// Applicable to Windows
-	ConsoleSize [2]int    // Initial console size
+	ConsoleSize [2]uint   // Initial console size
```

Comment:
Actually, thinking about this, I don't have to deal with back compat on Windows yet. I'll change it to a structure.